lib/lndclient/LndClient.ts

```diff
@@ -72,6 +72,22 @@ class LndClient extends BaseClient {
     });
   }
 
+  public connect = async () => {
+    this.logger.info(`trying to connect to lnd with uri: ${this.uri}`);
+    this.lightning = new LightningClient(this.uri, this.credentials);
+
+    try {
+      const getInfoResponse = await this.getInfo();
+      if (getInfoResponse) {
+        this.setStatus(ClientStatus.CONNECTION_VERIFIED);
+        this.subscribeInvoices();
+      }
+    } catch (err) {
+      this.logger.error(`could not fetch info from lnd host, error: ${err}, retrying in 5000 ms`);
```

Comment:
You should use `JSON.stringify(err)` else `could not fetch info from lnd host, error: [object Object], retrying in 5000 ms` will get logged.

Reply:
fixed the it test as well :)